build-system/server/app-index/template.js

```diff
@@ -78,12 +78,14 @@ function renderTemplate(opt_params = {}) {
     TopLevelWrap(ProxyForm({htmlEnvelopePrefix})),
     TopLevelWrap(FileList({basepath, htmlEnvelopePrefix, fileSet})),
 
+    /* eslint-disable local/no-forbidden-terms */
     html`
       <div class="center">
         Built with 💙 by
         <a href="https://ampproject.org" class="underlined">the AMP Project</a>.
```

Comment:
Would changing it to `amp.dev` fix it?

Reply:
It would. and since ampproject.org redirects to amp.dev anyways, that makes sense. ill make the change